examples/deepcopy-gen/generators/deepcopy.go

```diff
@@ -642,7 +681,7 @@ func (g *genDeepCopy) doMap(t *types.Type, sw *generator.SnippetWriter) {
 			sw.Do("for key := range *in {\n", nil)
 			sw.Do("(*out)[key] = struct{}{}\n", nil)
 			sw.Do("}\n", nil)
-		case t.Elem.IsAssignable():
+		case !hasPostDeepCopyMethod(t.Elem) && t.Elem.IsAssignable():
```

Comment:
Does this mean the existence of the (private) method inhibits copying?  Why do we need the tag, then?  Shouldn't those be orthogonal concepts?